src/app/screens/onboarding/detail-form/detail-form.component.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Inside the component folder you don't have to write code from HTML tag. Just need to write code specific to that div. Remove css as well and use tailwind

Reply:
Done sir i will upload the new changes now